lib/attributor/types/class.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../exceptions'
+
+module Attributor
+  class Class
+    include Type
+
+    def self.native_type
+      return ::Class
+    end
+
+    def self.load(value, context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
+      unless value.kind_of?(::String) || value.nil?
+        raise IncompatibleTypeError,  context: context, value_type: value.class, type: self
+      end
+
+      value = "::" + value if value[0..1] != '::'
+
+      value && eval(value)
```

Comment:
Never, ever eval code that comes potentially from a user please.
That needs to be changed to do something equivalent to ActiveSupport's constantize (i.e. a recursive const_get ...)